Gemfile

```diff
@@ -2,7 +2,7 @@
 
 source 'https://rubygems.org'
 
-gem 'rails'
+gem 'rails', ENV['RAILS_VERSION'] || '~> 7.0'
```

Comment:
Locally, the default Rails version is 7.0 while the default Solidus version is master.